### PR TITLE
Add Bundle files and generation

### PR DESCRIPTION
**Pull request by lucacome**
### Proposed changes
Added files and updated Make target to follow new [docs](https://redhat-connect.gitbook.io/certified-operator-guide/appendix/bundle-maintenance-after-migration)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,12 +26,13 @@ generate-metadata: generate-crds
...
-	cp deploy/olm-catalog/nginx-ingress-operator/nginx-ingress-operator.package.yaml bundle/
-	./hack/copy_manifests.sh
-	-rm bundle.zip
-	zip -j bundle.zip bundle/*.yaml
-
-.PHONY: build
\ No newline at end of file
+	@mkdir bundle/$(TAG)
+	@cp deploy/crds/* bundle/$(TAG)
+	@cp deploy/olm-catalog/nginx-ingress-operator/nginx-ingress-operator.package.yaml bundle/
```
Comment (mikestephen) [inline Makefile:31]: Is it intentional that this line copies to `bundle` and not `bundle/$(TAG)` like the others?

**Review by mikestephen — APPROVED** [review]
Looks ok to me, but I'm not very familiar with this.

**Review by Dean-Coakley — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (lucacome) [reply Makefile:31]: yeah it's just needed once in the root folder

Comment (mikestephen) [reply Makefile:31]: thanks, looks good

**Pull request merged**